### PR TITLE
Add support for reading DepedencyContext from deps file

### PR DESCRIPTION
**Pull request by pakrym**
@davidfowl

### CODE REVIEW

File: test/Microsoft.Extensions.DependencyModel.Tests/DependencyContextCsvReaderTests.cs [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+using System.Text;
+using Microsoft.Extensions.DependencyModel;
+using FluentAssertions;
+using Xunit;
+
+namespace Microsoft.Extensions.DependencyModel.Tests
+{
+    public class DependencyContextCsvReaderTests
+    {
+
```
Comment (davidfowl) [inline test/Microsoft.Extensions.DependencyModel.Tests/DependencyContextCsvReaderTests.cs]: nit: extra space

**Comment by davidfowl** [comment]
Add a single functional tests

**Comment by pakrym** [comment]
Added four

**Comment by davidfowl** [comment]
:shipit:

File: scripts/dotnet-cli-build/TestTargets.cs [hunk 2]
```diff
@@ -149,13 +118,10 @@ public static BuildTargetResult RestoreTests(BuildTargetContext c)
...
         {
             var dotnet = DotNetCli.Stage2;
-
-            var configuration = c.BuildContext.Get<string>("Configuration");
-
             foreach (var testProject in TestProjects)
             {
                 c.Info($"Building tests: {testProject}");
-                dotnet.Build("--configuration", configuration)
+                dotnet.Build()
```
Comment (davidfowl) [inline scripts/dotnet-cli-build/TestTargets.cs]: ?

**Comment by pakrym** [comment]
@dotnet-bot test Windows_NT Debug Build please
@dotnet-bot test Windows_NT Release Build please

**Comment by pakrym** [comment]
@dotnet-bot test Windows_NT Debug Build please
@dotnet-bot test Windows_NT Release Build please

**Comment by pakrym** [comment]
@dotnet-bot test Windows_NT Debug Build please
@dotnet-bot test Windows_NT Release Build please

**Comment by davidfowl** [comment]
zomg

**Comment by pakrym** [comment]
??

**Comment by davidfowl** [comment]
The windows builds.

**Comment by pakrym** [comment]
I'm crying, have 4 pr's ready to merge

**Comment by pakrym** [comment]
@dotnet-bot test Windows_NT Debug Build please
@dotnet-bot test Windows_NT Release Build please

**Pull request merged**